### DISCUSSION TITLE
Updating ASLprep to version 5.0.0

### DISCUSSION THREAD

**Discussion by lucsanara** [neurodesk/neurodesk.github.io-hugo#381] [Q&A] [answered]
Hello!!!
Could anyone update aslprep to version 0.5.0? It would be really useful for us to process pcasl images including m0
Thank you in advance!!

**Comment by stebo85** [comment, marked as answer]
Dear @lucsanara ,

you can do it yourself on GitHub through a simple pull request against this file: 
https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/aslprep/build.sh

Just click edit in GitHub, change the version number and send the pull request 

Thank you 
Steffen

**Comment by stebo85** [comment]
Dear @lucsanara,

aslprep is now available in Neurodesk in version 0.5.0 :) Until we release a new Neurodesk version, you need to click the Update button in the Application menu to make the new software visible:

<img width="1030" alt="image" src="https://github.com/NeuroDesk/neurodesk.github.io/assets/4021595/80bcc6ed-4f22-4b26-8991-938ab0d508cf">

Let us know if everything works as expected and thank you for the pull request!

Steffen

**Comment by lucsanara** [comment]
Thank you!! Hope it works good
